topydo/ui/Main.py

```diff
@@ -449,6 +491,20 @@ def _console_width(self):
 
         return sz
 
+    def _is_pending(self):
+        if len(self.pending_todos) > 0:
+            return True
+        else:
+            return False
+
+    def _pending_todos_handler(self, p_todo_id):
```

Comment:
This function does two things, but that's not apparent from its name (again, the verb thing I mentioned somewhere else earlier). I suspect that this function should be split and do only one thing.